commands/psp-publish.js

```diff
@@ -214,10 +214,10 @@ async function upsertConfigData (j1Client, config, templateData, section) {
       displayName: item.name,
       adopted: item.adopted,
       provider: item.provider,
+      summary: item.summary,
       type: item.type,
       createdOn: (new Date()).getTime(),
-      updatedOn: (new Date()).getTime(),
-      reviewedOn: -1
```

Comment:
Why are we removing `reviewedOn`? I'm not familiar with its original use, but I thought I'd ask.

Reply:
this would always set `reviewedOn` to 1969... it would overwrite the reviewedOn date even if it was updated properly by another means